dir.targets

```diff
@@ -87,6 +88,13 @@
     </BinPlaceConfiguration>
   </ItemGroup>
 
+  <!-- For tests we want to separate the bin directory and the working directory to have a working directory per build configuration. -->
+  <ItemGroup Condition="'@(BinPlaceConfiguration)' == '' AND '$(BinPlaceTest)' == 'true'">
```

Comment:
There is no need to seperate this out.  You're already applying the BinPlaceTest condition on the TestWorkingPath in the binplace targets so this is redundant.

Reply:
Indeed. Thanks.

Reply:
Also please consider reusing the $(TestPath) property to ensure that these two paths don't get out of sync.